### PR TITLE
Adds bio-bags, primarily for xenobiologists.

### LINKED ISSUE #11765: Slime processor annoying mass clicking
When you're grinding slimes, if you have steroided them, and upgraded the processor you have a million slime cores you need to manually click into the fridge, and its really annoying since you go through a lot of slimes in xenobio. There should be a way to like set the processor to autofeed to an adjacent fridge so you don't need to click

### PR DESCRIPTION
**Pull request by harmonyn**
Fixes https://github.com/tgstation/-tg-station/issues/11765.

Bio-bags: like mining satchels, trash bags, etc., but for the storage and disposal of biohazardous waste. 

biobag

What this means for you: No more clicking on over 9000 used slime extracts to put them into the trash! They can hold some other stuff too, mostly medical-related, but obviously this is the big one. Storage capacity is 25. This is less than trash bags.

One biobag is included in each L3 biohazard closet. Xenobio has two of these that they rarely use. So does virology.

### CODE REVIEW

**Comment by Fox-McCloud** [comment]
pics?

**Comment by bgobandit** [comment]
added

**Comment by Fox-McCloud** [comment]
I can understand it holding tons and tons and tons of slime cores, but it being able to hold 50 beakers or syringes and it still being a weight class 1 item seems a bit ridiculous.

**Comment by Bawhoppen** [comment]
This is a fix for this https://github.com/tgstation/-tg-station/issues/11765

**Comment by Bawhoppen** [comment]
Also @Fox-McCloud Trashbags and chem bags (maybe) can already do that

**Comment by KorPhaeron** [comment]
Needs a changelog

File: code/game/objects/items/weapons/storage/bags.dm [hunk 1]
```diff
@@ -10,6 +10,7 @@
  *		Plant Bag
  *		Sheet Snatcher
  *		Book Bag
+ *      Biowaste Bag
```
Comment (Giacom) [inline code/game/objects/items/weapons/storage/bags.dm]: Muh indention.

File: code/game/objects/items/weapons/storage/bags.dm [hunk 2]
```diff
@@ -359,3 +360,19 @@
...
+/obj/item/weapon/storage/bag/bio
+	name = "biowaste bag"
+	icon = 'icons/obj/chemical.dmi'
+	icon_state = "biobag"
+	desc = "A bag for the safe disposal of biowaste."
+	storage_slots = 50
+	max_combined_w_class = 200
+	w_class = 1
+	preposition = "in"
+	can_hold = list(/obj/item/slime_extract, /obj/item/weapon/reagent_containers/syringe, /obj/item/weapon/reagent_containers/glass/beaker, /obj/item/weapon/reagent_containers/glass/bottle, /obj/item/weapon/reagent_containers/blood, /obj/item/weapon/reagent_containers/hypospray/medipen)
```
Comment (CosmicScientist) [inline code/game/objects/items/weapons/storage/bags.dm]: Would it be useful to include monkey cubes too?

Comment (bgobandit) [inline code/game/objects/items/weapons/storage/bags.dm]: useful yes, idk if I can justify it with the current fluff though

File: code/game/objects/items/weapons/storage/bags.dm (hunk 1 above)
Comment (bgobandit) [inline code/game/objects/items/weapons/storage/bags.dm]: it looks right in the ide :^)

**Comment by Bawhoppen** [comment]
wait, this is bioWASTE?

**Comment by bgobandit** [comment]
the primary use I am envisioning is for xenobiologists to be able to mass dispose of the 100000000000 used slime extracts they produce, yes

**Comment by Bawhoppen** [comment]
What about putting the 100000000000000^10000 slime extracts they PRODUCE into the slime fridge?

**Comment by bgobandit** [comment]
that is undocumented functionality :^)

**Comment by Bawhoppen** [comment]
Just consider their primary function for not being WASTE, but then let you do that too

**Comment by bgobandit** [comment]
but they CAN do that already, they take all children of slime extract

**Comment by Bawhoppen** [comment]
I know, codewise it would be the same, but NAME WISE

**Comment by bgobandit** [comment]
fiiiine

**Comment by freerealestate** [comment]
being able to carry around like 100 syringes and 50 beakers in your pocket is probably not the best idea

**Comment by bgobandit** [comment]
but that is already possible

**Comment by freerealestate** [comment]
I just checked, chemistry bags don't carry syringes.

**Comment by bgobandit** [comment]
trash bags do

**Comment by freerealestate** [comment]
they only have 30 slots, afaik boxstation only starts with the one and janitors don't have access to syringes or syringe guns by default

**Comment by bgobandit** [comment]
sure fine ok

storage capacity is now 25

**Comment by freerealestate** [comment]
i think removing syringes would've been the better option tbh
you can still run around with like 13 beakers and 12 syringes or whatever in your pocket, just endlessly reloading a syringe gun if you want

**Comment by freerealestate** [comment]
Does no one else think it'll become a balance issue for pro chemists to be able to fill this with 6 beakers and 19 syringes (or 3/22 with large beakers), hide it in their pocket and run around with a syringe gun shooting everyone full of poison?

Previously you could only hold 7 syringes per inventory space at a time, in items in your bag or on your belt. It's a buff to the poison kit, for sure.

**Comment by Bawhoppen** [comment]
Not when you can already do it AND you have to manually sort through each different beaker.

**Comment by freerealestate** [comment]
We already established you couldn't already do it. 

Don't think 'click a syringe, a beaker three times, then your syringe gun' counts as 'manually sorting through each beaker,' whatever you mean by that.

**Comment by Bawhoppen** [comment]
Like chem bags where you must sort through each pill, you have to do the same with reagent containers such as beakers.   And you most definitely already can.

**Comment by freerealestate** [comment]
What item could you do it with?

I already established you only needed 6 regular sized beakers to fill 19 syringes, so sorting through them wouldn't be a problem.

**Comment by Bawhoppen** [comment]
Trashbag

Bluespace variety for a shitload of storage
ss 2015-10-05 at 09 15 10

Its a very good belt.

**Comment by freerealestate** [comment]
Good point, except it's high-end research item that (afaik) only goes on your belt slot, or in a bag of holding.

Multiple copies of these spawn at roundstart and fit in your pocket.

**Comment by Bawhoppen** [comment]
normal trashbags hold quite a few too

**Comment by freerealestate** [comment]
They only have a max combined weight class of 30 (these have 200), only one spawns at roundstart on Box (afaik) and they're only accessible from the belt slot.

**Pull request merged**